charts/template-api/Chart.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm base-chart for Kubernetes to deploy the api
+name: template-api
```

Comment:
The name is really important here, thinks generic and reusable. I suggest `symfony`. It will be available like this `epourail/symfony`